static/usr/lib/systemd/system/vdsm-network.service.in

```diff
@@ -8,6 +8,8 @@ After=libvirtd.service openvswitch.service NetworkManager.service
 Type=oneshot
 EnvironmentFile=-/etc/sysconfig/vdsm
 ExecStartPre=@BINDIR@/vdsm-tool dump-bonding-options
+ExecStart=@BINDIR@/vdsm-tool nwfilter
+ExecStart=@BINDIR@/vdsm-tool dummybr
 ExecStart=@BINDIR@/vdsm-tool restore-nets
```

Comment:
`restore-nets` needs to be last as it takes care of restoring host networking if the current state does not match the desired.